configs/mainnet.yaml

```
@@ -64,6 +64,9 @@ EIP7441_FORK_EPOCH: 18446744073709551615
 # EIP7732
 EIP7732_FORK_VERSION: 0x09000000  # temporary stub
 EIP7732_FORK_EPOCH: 18446744073709551615
+# EIP7805
+EIP7805_FORK_VERSION: 0x10000000  # temporary stub
```

One more little suggestion, we can/should do 0x0a here instead 😄
```suggestion
EIP7805_FORK_VERSION: 0x0a000000  # temporary stub
```

---

ohhhh my bad. Nice catch!